src/interactive-config.coffee

```diff
@@ -48,10 +53,15 @@ interactiveConfig.prompt = (config = {}, callback) ->
     name: "apiaryApiName"
     message: "Please enter Apiary API name"
     default: config['custom']?['apiaryApiName']
-    when: (answers) -> (
-      (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
-      (answers['apiary'] == true && answers['apiaryApiKey'] != '')
-    )
+    when: (answers) ->
+      # it's not GA on the Apiary side yet
+      if process.env['APIARY_BETA']?
+        (
+          (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
+          (answers['apiary'] == true && answers['apiaryApiKey'] != '')
+        )
```

Comment:
`answers.apiary is true && answers.apiaryApiKey && config.custom?.apiaryApiName` ?